aeronautical/asterix/fixed.ksy

```diff
@@ -0,0 +1,47 @@
+meta:
+  id: fixed
```

Comment:
We can't really have these generic names, as they will be repo-global. Please consider something like `asterix_fixed`?

Reply:
This will require changing explicit, repetitive, extended and compound types also. And will give a bunch of boiler plate asterix_ all over the category catalogs and definitions. But If this is an impediment the I'll change it.

Reply:
Hang on, we'll need to (1) resolve the license question first, and once/if we'll do that, then (2) my next question will be "what is the value in keeping this as many .ksy files?"

I mean, there *is* a namespacing mechanism. For example, you do a single top-level type called `asterix` and then all types inside it will be namespaced:

```yaml
meta:
  id: asterix
types:
  fixed: {}
  repetitive: {}
  extended: {}
  compound: {}
```

From the outside world, these will be seen as `Asterix.Fixed`, `Asterix.Repetitive`, etc. From the inside of the file, you can still refer to them as `fixed`, `repetitive`, etc.

Reply:
Well, it is a file for a type, don't understand why it is bad since in general this is preferable. In any case. I could have an asterix file, and define message, block, record, and also fixed, repetitive extended and compound. But then Asterix will import cat_002 (similar to asterix_record.ksy) , which in turn imports cat_002_catalog, which in turn need to import asterix, to have access to fixed, repetitive etc. ¿Is this supported? because I get an error when doing this circular reference imports.